app/src/handlers/census/DeathEventHandler.ts

```diff
@@ -57,6 +53,18 @@ export default class DeathEventHandler implements EventHandlerInterface<DeathEve
             return false;
         }
 
+        try {
+            this.aggregateHandlers.map(
+                (handler: EventHandlerInterface<DeathEvent>) => void handler.handle(event),
+            );
+        } catch (e) {
```

Comment:
This will never catch as handle returns a promise. If you want a catch this you van to wrap `map` into `Promise.all` and then call `await` before it. However this means that if one fails the will all return as failed. Instead I would handle then all seperatly with `handle(event).catch((e) => logger.info('Jup something went wrong'))`